### ISSUE TITLE
What you get when you press right click

### ISSUE THREAD

**Issue by selrashidi** [layersoflondon/application#101]
One of the things that needs to be dealt with is what we download/save/print when we press the right click button. Currently, a small segment of the map is downloaded/printed. We need to think of what this should be in the final phase, as what we are able to download will have a strong impact on the usefulness of the website. Being able to download/print information being a major advantage.

**Comment by edtjones** [comment]
Good suggestion. I think we should build proper functions for download and print in phase 2.

--
Sent from my phone: please forgive the brevity.

Ed Jones

**Comment by mpdavies** [comment]
I agree. It is partly also about making resources available for re-use. We can put some limits on this, partly in terms of files size etc, but no reason why someone could not download at least what fits within the viewable area.

**Comment by Addaci** [comment]
I anticipate that people will wish to inspect one geographic area and will wish to toggle or slide between layers. They may then wish to download a shape defined area (oblong, or more complex shape) to their own server or computer for further analysis. Gabor Bakos, one of our hackers, is working with me on Hack 9: cartopop. He has knocked together a quick and dirty Leaflet based frame to look at the London Bridge to Woolwich oblong for maps between 1682 and 1896.http://www.fragosus.pe.hu/Thames/RumseyCollection.htm

Already we are thinking how we can modify this for a tighter frame to use in our Deptford Creek  soundscape hack, which requires us to assemble as many detailed Deptford plans and maps as we can find between 1600 and 2017 and then to georectify them as precisely as we can, so that we can have geolocated soundposts on a 1650 map and then do sound sampling from exactly the same geolocated soundposts in 2017. [https://github.com/orgs/Maphackathon/projects/7]
